File: JavaScript/8js 1/getASCII.js
/*
|--------------------------------------------------------------------------
| get ascii value of character
|--------------------------------------------------------------------------
|
*/

function getASCII(c) {
   return c.charCodeAt(0);
}


getASCII = (c) => c.charCodeAt();


function getASCII(c) {
   let a =
      " !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~";
   return a.indexOf(c) + 32;
}


console.log(getASCII("A"), 65);
console.log(getASCII(" "), 32);
console.log(getASCII("!"), 33);

File: Python/8py 1/get_ascii.py
"""
|--------------------------------------------------------------------------
| get ascii value of character
|--------------------------------------------------------------------------
|
"""

def get_ascii(c):
   return ord(c)


get_ascii = ord


def get_ascii(c):
   return bytearray(c.encode('ascii'))[0]


print(get_ascii("A"), 65)
print(get_ascii(" "), 32)
print(get_ascii("!"), 33)